AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/README.md

```diff
@@ -0,0 +1,74 @@
+## DataStore with Auth Integration Tests
+
+The following steps demonstrate how to set up an GraphQL endpoint with AppSync. 
+The auth configured will be Cognito UserPools. 
```

Comment:
We can probably combine these two sentence into one:

The following steps demonstrate how to setup a GraphQL endpoint with AppSync and Cognito User Pools.